CHANGELOG.md

```diff
@@ -1,9 +1,31 @@
 # Changelog
 
-The latest published Druid release is [0.6.0](#060---2020-06-01) which was released on 2020-06-01.
-You can find its changes [documented below](#060---2020-06-01).
+he latest published Druid release is [0.7.0](#070---2020-12-31) which was released on 2020-06-01.
```

Comment:
"The"

Reply:
Thanks, I did this with half my brain yesterday. And I also aspired to get it merged by last night. alas.

Reply:
No worries. I've meant "2020-06-01".